docs/design/assignment.md

```diff
@@ -0,0 +1,329 @@
+# Assignment
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Overview](#overview)
+-   [Syntax](#syntax)
+-   [Simple assignment semantics](#simple-assignment-semantics)
+-   [Built-in types](#built-in-types)
+-   [Tuples, structs, and classes](#tuples-structs-and-classes)
+-   [Extensibility](#extensibility)
+    -   [Simple assignment](#simple-assignment)
+    -   [Arithmetic](#arithmetic)
+    -   [Bitwise and bit-shift](#bitwise-and-bit-shift)
+    -   [Defaults](#defaults)
+-   [Alternatives considered](#alternatives-considered)
+-   [References](#references)
+
+<!-- tocstop -->
+
+## Overview
+
+Values can be assigned to variables using the `=` operator:
+
+```
+var a: i32 = 5;
+a = 6;
+```
+
+For each binary [arithmetic](expressions/arithmetic.md) or
+[bitwise](expressions/bitwise.md) operator `$`, a corresponding compound
```

Comment:
I like the usage of "`$`" and "`$=`" below, but I think "`$`" is a distraction here because there is no accompanying "`$=`".

Reply:
Good point, added a `$=` here.